system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -1925,28 +1925,33 @@ class CLdapClient : implements ILdapClient, public CInterface
             StringBuffer hostbuf;
             int rc = LDAP_SERVER_DOWN;
             char *ldap_errstring=NULL;
-            for(int retries = 0; retries <= LDAPSEC_MAX_RETRIES; retries++)
+            for (int numHosts=0; numHosts < m_ldapconfig->getHostCount(); numHosts++)
```

Comment:
`numHosts` as an incrementing counter name is confusing. `hostNum` or `hostIdx` seem more intuitive. The method call is probably low overhead, but you might consider initializing to the count and decrementing to a terminal condition instead of calling the function multiple times.

Reply:
This code is similar to code in other spots, so I left the conventions the same.